educe/stac/edit/cmd/nudge_dialogue.py

```diff
@@ -114,14 +114,14 @@ def _nudge_dialogue(doc, tid, direction):
     """
     prev_turn, turn, next_turn = \
         _window1(lambda x: st.turn_id(x) == tid,
```

Comment:
Looking at `_window1` just above, this function could use a small fix of the docstring (match is the item matching the predicate) and a small fix in the code (`before = [None] + now[:-1] ̀).
I cannot comment on lines outside the "diff zone".